### ISSUE TITLE
Elixir Facet SDK is not defined since Elixir 1.11 (with possible solution)

### ISSUE THREAD

**Issue by ruudk** [KronicDeth/intellij-elixir#1832]
**Describe the bug**
After upgrading to Elixir 1.11 PHPStorm shows a bar `Elixir Facet SDK is not defined` that won't go away.

I never had this issue before Elixir 1.11. 

When I click `Setup Elixir Facet SDK` it shows me this screen:
<img width="1111" alt="Screenshot 2020-10-08 at 09 52 44@2x" src="https://user-images.githubusercontent.com/104180/95430430-07ebf200-094c-11eb-9a7d-f2acc91b327c.png">

When I change it to Elixir 1.11 and save it, the banner goes away and everything is fine.

But when I close the project and open it again, the same problem happens.

**Desktop:**
 - OS: macOS
 - Version 10.15.7 (19H2)

**Erlang:**
- Installer homebrew
- Version 23

**Elixir:**
- Installer homebrew
- Version 1.11

**Plugin:**
- Version 11.8.1

PhpStorm 2020.3 EAP
Build #PS-203.4203.36, built on October 1, 2020

**Comment by ruudk** [comment]
I made a backup of my `.idea` directory and then deleted the `.idea` folder.

When I restarted PHPStorm it created a new `.idea` folder so that I could make a diff between the new and the old one.

I found out that `${projectName}.iml` file holds this config that no longer exists:
```xml
    <facet type="Elixir" name="Elixir">
      <configuration sdkName="Elixir 1.10.4" />
    </facet>
```

So I suspect that that causes the issue. The config references a sdk that no longer exists. Somehow the GUI doesn't overwrite it when you make changes to it.

@KronicDeth Hope this helps
